src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -494,6 +498,21 @@ export class ZaakViewComponent extends ActionsViewComponent implements OnInit, A
         });
     }
 
+    private openZaakHeropenenDialog(): void {
+        this.websocketService.doubleSuspendListener(this.zaakListener);
```

Comment:
Is deze suspend niet iets te vroeg, nu moet je binnen 5 seconden het dialoog venster sluiten, anders mis je hem. kan deze niet beter in een pipe op heropenen